### DISCUSSION TITLE
Labeling polygon out of the image

### DISCUSSION THREAD

**Discussion by zhanxinrui** [wkentaro/labelme#1075] [Ideas / Feature requests]
Could I use labelme to annotate the polygon points that are out of the image, which means (x, y) do not belong to  [0, W] and [0, H]. 
However, it seems the cursor is not working when it is out of the image area. I don't want to add padding to the image before labeling and then remove them, is there an easy way? Thanks a lot!

**Comment by jiwkim9699** [comment]
edit canvas.py file
### original
```
    def outOfPixmap(self, p):
        w, h = self.pixmap.width(), self.pixmap.height()
        return not (0 <= p.x() <= w - 1 and 0 <= p.y() <= h - 1)
```
### changed
```
    def outOfPixmap(self, p):
        w, h = self.pixmap.width(), self.pixmap.height()
        return False
```
